clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/validators/loadbalancer/UpsertOpenstackLoadBalancerAtomicOperationValidator.groovy

```diff
@@ -29,6 +29,9 @@ import org.springframework.validation.Errors
 @Component
 class UpsertOpenstackLoadBalancerAtomicOperationValidator extends AbstractOpenstackDescriptionValidator<OpenstackLoadBalancerDescription> {
 
+  static final int MIN_PORT = 1
+  static final int MAX_PORT = (1 << 16) - 1
```

Comment:
Is this the right place for this? I feel like security groups are defined with a port as well, and will likely need this constant as well.